libraries/common/io.effekt

```diff
@@ -3,15 +3,15 @@ module io
 import ref
 import queue
 
-extern include llvm "../llvm/io.ll"
-extern include js "../js/io.js"
-
-
 // Event Loop
 // ----------
 
 type Task[T] = () => T at {io, async, global}
 
+extern llvm """
+  declare void @c_timer_start(%Int, %Stack)
+"""
```

Comment:
This wouldn't be duplicated, if instead we define a function
`c_spawn(%Stack)` that just calls c_timer_start with `0`.